### ISSUE TITLE
CTRL+SHIFT+O to open rainway overlay keeps showing

### ISSUE THREAD

**Issue by Knagie** [RainwayApp/bug-tracker#320]
# CTRL+SHIFT+O to open Rainway overlay keeps showing

I don't remember if it also was staying there in the previous versions
It does bother me now when I tried to play Cube World for the first time.

## Steps

- Open a game through Rainway.
- Notice the small overlay that says "Press Ctrl_Shift_O to open the Rainway overlay".
- Try and get this closed/moved.

## What happens

The small overlay (See screenshot) keeps showing over the game while playing. And it sometimes covers parts that are needed, like subtitles and in case of the game Cube World it covers the text field to add the name of the character

![image](https://user-images.githubusercontent.com/7049201/56079230-ef038e80-5df1-11e9-89a4-560fee59359d.png)
_(The black bar at the bottom is the taskbar of my client machine which overlapped with the Rainway screen)_

## What is expected

That it would disappear after a while or that it was able to be dragged around to move to a less obstructing part of the screen. Or make it a button you can click in a corner for example if you do want it showed.

## Host

- OS: Windows 10 Pro
- Version: 1809 17763.437
 
## Browser

- OS: Windows 10 Education
- Version: 1809 17763.379
- Browser: Chrome
- Browser Version: 73.0.3683.86

**Comment by BetaLeaf** [comment]
Have you tried pressing CTRL+SHIFT+O to show and then again to hide the overlay and the message box?

**Comment by Knagie** [comment]
Yes I did and it kept showing. Funny thing is that now for some reason it doesn't show when starting a game, I only enabled the remote desktop option and it just stopped showing on all games, also ones I didn't start yet. 
So my problem is solved, not sure if I should close this or if it's indeed a bug more people have.

**Comment by BetaLeaf** [comment]
I'd keep it open. It's unintential the information popup doesn't
automatically dismiss itself.
